conf/minifi.properties

```diff
@@ -20,6 +20,9 @@ nifi.administrative.yield.duration=30 sec
 # If a component has no work to do (is "bored"), how long should we wait before checking again for work?
 nifi.bored.yield.duration=100 millis
 
+# Comma separated path for the extension libraries. Relative path is relative to the minifi executable.
+nifi.extension.path=../extensions/*
```

Comment:
the extensions are still essentially part of the agent, the MINIFI_HOME, as I understand, could be moved to anywhere on the system, I would like them (extensions) not to have to be moved with, just as we don't have to move the executable

(note: we can still use the ${MINIFI_HOME} variable in the path if need be just as with any other property)

Reply:
I currently see two usual ways of launching minifi:
- The normal installation, where the MINIFI_HOME is usually /opt/minifi, and the binary is under MINIFI_HOME/bin
- After compilation, launching the compiled binary in a preexisting MINIFI_HOME

In the first case, either version is fine. In the second case, we would need extensions to be installed to build/extensions/ for them to work without extra effort, or we could load them from MINIFI_HOME/extensions/ in the proposed case.

Another use case that I could see happening in the future is standard linux packages:
- minifi binary under /usr/bin/minifi
- config files under /etc/minifi
- extensions under /usr/share/minifi/extensions, and I imagine MINIFI_HOME would be /usr/share/minifi.

I think the default would cover more use cases with a path relative to MINIFI_HOME, and we could avoid the "../" at the beginning of practically every path pattern. Given the above points, do you still prefer paths relative to the binary?

Reply:
regarding the second point: you would need to move the compiled extensions to `MINIFI_HOME/extensions`, I wouldn't call that "without extra effort", but you could specify `nifi.extension.path=*` and it would use the freshly built extensions while `${MINIFI_HOME}/extensions/*` would achieve the other use-case

I don't think we can reference the `EXE_DIR` from the configuration file, so if someone decides to use the extension relative to the exe they would have to hard code the exe dir